config.app.js

```diff
@@ -51,5 +54,8 @@ module.exports = {
   PASSWORD_MIN_LENGTH: integerFromText(__PASSWORD_MIN_LENGTH__, 8),
   PASSWORD_MAX_LENGTH: integerFromText(__PASSWORD_MAX_LENGTH__, 72),
   ABOUT_MAX_LENGTH: integerFromText(__ABOUT_MAX_LENGTH__, 256),
-  I18N_ENABLED: booleanFromText(__I18N_ENABLED__, false)
+  I18N_ENABLED: booleanFromText(__I18N_ENABLED__, false),
+  ALLOW_CHANGE_EMAIL: booleanFromText(__ALLOW_CHANGE_EMAIL__, true),
+  ALLOW_CHANGE_PASSWORD: booleanFromText(__ALLOW_CHANGE_PASSWORD__, true),
+  ALLOW_CHANGE_LANG: booleanFromText(__ALLOW_CHANGE_LANG__, true)
```

Comment:
Do not use variable for changing language. It will basically use the I18N_ENABLED